keyboards/cannonkeys/sagittarius/sagittarius.c

```diff
@@ -0,0 +1,19 @@
+#include "sagittarius.h"
+
+void keyboard_pre_init_kb(void) {
+    setPinOutput(INDICATOR_PIN_0);
+    setPinOutput(INDICATOR_PIN_1);
+    setPinOutput(INDICATOR_PIN_2);
+
+    keyboard_pre_init_user();
+}
+
+bool led_update_kb(led_t led_state) {
+    bool runDefault = led_update_user(led_state);
+    if (runDefault) {
+      writePin(INDICATOR_PIN_0, led_state.num_lock);
+      writePin(INDICATOR_PIN_1, led_state.caps_lock);
+      writePin(INDICATOR_PIN_2, led_state.scroll_lock);
+    }
+    return runDefault;
+}
```

Comment:
In theory, shouldn't be needed anymore with the recent addition of:
https://docs.qmk.fm/#/feature_led_indicators?id=configuration-options

Reply:
I'd rather stick with this known working implementation. I see no reason why I should be forced to move away from this approach.

Reply:
This simply moves the code into core, rather than in the keyboard or keymap code.  It still allows for the same functionality.  In fact, changing it means less code that *you* have to maintain.  

Additionally, because this isn't a custom implementation, all you're doing is dumping tech debt onto us.